salt/utils/sdb.py

```diff
@@ -63,3 +63,31 @@ def sdb_set(uri, value, opts):
 
     loaded_db = salt.loader.sdb(opts, fun)
     return loaded_db[fun](query, value, profile=profile)
+
+
+def sdb_delete(uri, opts):
+    '''
+    Delete a value from a db, using a uri in the form of ``sdb://<profile>/<key>``. If
+    the uri provided does not start with ``sdb://`` or the value is not successfully
+    deleted, return ``False``.
+    '''
+    if not isinstance(uri, string_types):
```

Comment:
Why are we returning a string for one type of invalid argument but a bool for another?